src/world/Units/Unit.cpp

```diff
@@ -6288,3 +6288,66 @@ void Unit::jumpTo(Object* obj, float speedZ, bool withOrientation)
     float speedXY = getExactDist2d(x, y) * 10.0f / speedZ;
     getMovementManager()->moveJump(x, y, z, getAbsoluteAngle(obj), speedXY, speedZ, EVENT_JUMP, withOrientation);
 }
+
+#if VERSION_STRING == Cata
+DBC::Structures::MountCapabilityEntry const* Unit::getMountCapability(uint32_t mountType)
+{
+    if (!mountType)
+        return NULL;
+
+    auto const* mountTypeEntry = sMountTypeStore.LookupEntry(mountType);
+    if (!mountTypeEntry)
+        return NULL;
+
+    uint32_t zoneId, areaId = 0;
```

Comment:
Why do we initialize these two to 0 when we override it the next lines anyway?

Reply:
yeah was kinda useless you are true